### PR TITLE
Azure Monitor Exporter - Update codegen

### PR DESCRIPTION
**Pull request by rajkumar-rangaraj**
Ran autogen against new swagger and removed customized classes written to parse `duration` and stack frame `level`

### CODE REVIEW

**Review by reyang — APPROVED** [review]
LGTM.

File: sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs [hunk 1]
```diff
@@ -79,10 +79,10 @@ internal async ValueTask<int> AddBatchActivityAsync(Batch<Activity> batchActivit
             return response.Value.ItemsAccepted.GetValueOrDefault();
         }
 
-        private static TelemetryEnvelope GeneratePartAEnvelope(Activity activity)
+        private static TelemetryItem GeneratePartAEnvelope(Activity activity)
         {
             // TODO: Get TelemetryEnvelope name changed in swagger
-            TelemetryEnvelope envelope = new TelemetryEnvelope(PartA_Name_Mapping[activity.GetTelemetryType()], activity.StartTimeUtc);
```
Comment (TimothyMothra) [inline sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs:85]: I think you can remove this TODO.
Also, this variable should be renamed to match the type.

**Review by TimothyMothra — APPROVED** [review]

File: sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/autorest.md [hunk 2]
```diff
@@ -4,6 +4,6 @@ Run `dotnet msbuild /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    -  https://raw.githubusercontent.com/srnagar/swagger/d93c35263ce8b28fefdafd702adbb1a0e2347156/application-insights.json
+    -  https://github.com/srnagar/swagger/blob/05b8ab86a14164a364ac18869fa2b2e689324403/application-insights.json
 namespace: OpenTelemetry.Exporter.AzureMonitor
```
Comment (pakrym) [inline sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/autorest.md:8]: nit. You shouldn't need the namespace line.

Comment (rajkumar-rangaraj) [reply sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/autorest.md:8]: I will get this fixed in next PR.

**Pull request merged**